CODEOWNERS

```diff
@@ -457,7 +457,6 @@
 /include/display/                         @vanwinkeljan
 /include/dt-bindings/clock/kinetis_mcg.h  @henrikbrixandersen
 /include/dt-bindings/clock/kinetis_scg.h  @henrikbrixandersen
-/include/dt-bindings/dma/stm32_dma.h      @cybertale
```

Comment:
Please also remove references to #include <dt-bindings/dma/stm32_dma.h> in various drivers

Reply:
--> drivers/spi/spi_ll_stm32.c
--> drivers/i2s/i2s_ll_stm32.c
--> drivers/sperialuart_stm32.c
--> drivers/flash/flash_stm32_qspi.c